tests/02_eradiate/01_unit/scenes/measure/test_core.py

```diff
@@ -98,16 +86,45 @@ def test_mono_spectral_config(modes_all_mono):
     assert len(cfg.spectral_ctxs()) == 1
 
 
-@pytest.mark.parametrize("srf", ["data_store_srf", "local_file_srf"])
-def test_mono_spectral_config_srf(modes_all_mono, request, srf):
+def test_mono_spectral_config_srf(modes_all_mono, tmpdir):
     """
     A SRF is loaded from the data store/a local file.
     """
-    cfg = MeasureSpectralConfig.new(
-        wavelengths=[640.0, 650.0, 660.0],
-        srf=request.getfixturevalue(srf),
+    # existing prepared SRF
+    _ = MeasureSpectralConfig.new(
+        wavelengths=[650.0],
+        srf="sentinel_2a-msi-4",
     )
```

Comment:
Style + unnecessary assignment + assert something:
```suggestion
    assert MeasureSpectralConfig.new(wavelengths=[650.0], srf="sentinel_2a-msi-4")
```